examples/rest-grpc-multiplex/Cargo.toml

```diff
@@ -5,16 +5,17 @@ edition = "2021"
 publish = false
 
 [dependencies]
-axum = { path = "../../axum" }
+# needs to match tonic
+axum = { version = "0.7.5" }
```

Comment:
I wouldn't want examples within this repo to pull in a crates.io release of axum.

I'm considering moving examples to a separate repo anyways though, see #2942 for discussion.